nUpdate.Administration/UI/Dialogs/PackageAddDialog.cs

```diff
@@ -1508,6 +1519,113 @@ private void infoButton_Click(object sender, EventArgs e)
             updatingInfoDialog.ShowDialog();
         }
 
+        private void requirementsTypeComboBox_SelectedIndexChanged(object sender, EventArgs e)
+        {
+            requirementsTypeTabControl.SelectedIndex = requirementsTypeComboBox.SelectedIndex;
+        }
+
+        private void addRequirementButton_Click(object sender, EventArgs e)
+        {
+            UpdateRequirement _updateRequirement = null;
+            switch (requirementsTypeComboBox.SelectedIndex)
+            {
+                case 0:
+                    _updateRequirement = new UpdateRequirement(
+                        UpdateRequirement.RequirementType.osVersion,
+                        null, 
+                        osVersion(requiredOSComboBox.Text),
```

Comment:
Make that a ComboBox and use a dictionary containing the OS name and relating version.